js/components/DeploymentComponent.jsx

```diff
@@ -1,13 +1,15 @@
 /** @jsx React.DOM */
 
+var lazy = require("lazy.js");
```

Comment:
With capital L I am getting this eslint error, what is true regarded to the book I've mentioned:
```A function with a name starting with an uppercase letter should only be used as a constructor```
So I would like to stay with a lowercase l.